examples/graphql-service-query-complexity/graphql_service_query_complexity.metatags

```diff
@@ -0,0 +1,2 @@
+description: This example demonstrates securing a GraphQL service with SSL.
+keywords: ballerina, ballerina by example, bbe, graphql, service, security, ssl, tls
```

Comment:
is this description and the keywords are correct? might need to update them accordingly

Reply:
Oh, a copy-paste error. Will update it.

Reply:
Fixed with [8fcbe48](https://github.com/ballerina-platform/ballerina-distribution/pull/5530/commits/8fcbe4883a522dd6af5a5c0cc0a186672ad43d75)